src/CTA.Rules.Models/ProjectResult.cs

```diff
@@ -9,6 +9,7 @@ public class ProjectResult
         public List<PackageAction> UpgradePackages { get; set; }
         public List<PackageAction> ActionPackages { get; set; }
         public List<string> TargetVersions { get; set; }
+        public List<string> SourceVersions { get; set; }
```

Comment:
source version can just be string, since each project have only one framework version, but no harm in keep it

Reply:
Yeah I wanted to stay consistent, definitely agree with the fact they can both just be strings but I think Mark had ideas on how to use this in the future which is why he went with a list instead. I'll keep it like this to match targetVersions for now.